.npmrc

```diff
@@ -1 +1,2 @@
 package-lock=false
+legacy-peer-deps=true
```

Comment:
NPMv7+ automatically installs peer dependencies. However, Terra currently has in peer dependency versions which can cause install failures. This flag when set will utilize the legacy behavior of not installing peer dependencies in v6. This will be removed in a future PR when the conflicts are resolved.